docs/generators/aspnetcore.md

```diff
@@ -27,6 +27,8 @@ sidebar_label: aspnetcore
 |useNewtonsoft|Uses the Newtonsoft JSON library.| |true|
 |newtonsoftVersion|Version for Microsoft.AspNetCore.Mvc.NewtonsoftJson for ASP.NET Core 3.0+| |3.0.0-preview5-19227-01|
 |useDefaultRouting|Use default routing for the  ASP.NET Core version. For 3.0 turn off default because it is not yet supported.| |true|
+|enumNameSuffix|Suffix that will be appended to all enum names.| |Enum|
+|enumValueNameSuffix|Suffix that will be appended to all enum value names.| |Enum|
```

Comment:
What is the use case for this? Why wouldn't someone just put the expected value in the enum text field in the specification document?

Reply:
The behavior before my change is to append `Enum` to both the name of the enum type and each value.  So you end up with
```
enum MyEnum {
  Value1Enum,
  Value2Enum
}
```
I didn't want to break the existing default behavior, so I made that suffix customizable too.

Reply:
Sorry for the delay in response. The holidays were pretty hectic for me.

I think we'll want to call the value one `enumValueSuffix`. I'll do that in a separate commit.